js/load.js

```diff
@@ -17,6 +17,17 @@ import VueRegistry from './classes/VueRegistry.js';
 
 Vue.mixin(GlobalMixins);
 
+// adapt locale from $.pkp.app.currentLocale
+let momentLocale = $.pkp.app.currentLocale.toLowerCase();
+momentLocale = momentLocale.split('_')[0];
+momentLocale = momentLocale == 'en' ? 'en-gb' : momentLocale;
+const moment = require('moment');
+require('moment/locale/' + momentLocale);
+
+Vue.use(require('vue-moment'), {
+	moment,
+});
+
```

Comment:
This can be left out now since we're not including the `dateStatusModified` in the list itself. There will probably come a day where we may add moment.js, but I'd like to hold off if we can. We're already struggling with a very large bundle size and I'm not 100% sure the `require()` load system will work in all of our supported browsers, depending on how bable transposes that when building the JS bundle.

Reply:
I chose to use moment.js to have something like "2 days ago" which I think is a good presentation.
I will try to use vue-timeago which seems to be smaller

Reply:
It shouldn't be needed at all because of this comment: https://github.com/pkp/ui-library/pull/34#pullrequestreview-279479167

Reply:
Ok I remove vue-moment and rename param